core/services/fluxmonitorv2/validate.go

```diff
@@ -32,5 +35,29 @@ func ValidatedFluxMonitorSpec(ts string) (job.SpecDB, error) {
 	if specDB.SchemaVersion != uint32(1) {
 		return specDB, errors.Errorf("the only supported schema version is currently 1, got %v", specDB.SchemaVersion)
 	}
+
+	// Find the smallest of all the timeouts
```

Comment:
The only other validation the v1 FM job did was url validation, that now happens when we unmarshal the http tasks from the dot graph